examples/tutorials/SMPC_Protocols_Explanation.ipynb

```diff
@@ -0,0 +1,1181 @@
+{
```

Comment:
<blockquote>Does the compression also needs to be turned off, if so, why? (as in Théo's tuto)</blockquote><p>It makes the code faster, it's not compulsory</p><p> </p><blockquote>if resnet is in train() mode it can't run an encrypted forward pass</blockquote><p>Training resnet is not supported so far (and too expensive)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/4635/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Alright, thanks for the explanations! </p>